.eslintrc.cjs

```diff
@@ -224,11 +224,37 @@ module.exports = {
         },
         tailwindcss: {
           callees: tailwindCallees,
-          whitelist: ['light', 'hive-focus', 'hive-focus-within'],
+          whitelist: ['light', 'hive-focus', 'hive-focus-within', '_text-primary-600'],
```

Comment:
why this class has an underscore prefix?

Reply:
this link is not highlighted for some reason 
<img width="843" alt="image" src="https://github.com/user-attachments/assets/c32bd90a-68ab-4539-b69d-1ba9b006af41">

nextra uses `_` prefix for its classes